dtm0/it/all2all/README.md

```diff
@@ -43,3 +43,41 @@ it needs to be patched by the hare.patch in current directory
 to make it able to generate clusted configuration that includes
 DTM0 service. After that it should be built and installed into
 the system (see [README.md](https://github.com/Seagate/cortx-hare/blob/main/README.md) in the root of Hare repo).
+
+## Troubleshooting
+### 1. Test fails for starting motr processes
+If test fails at the stage where it start motr processes, please check if /etc/modprobe.d/lnet.conf has same interface as 
+the lnet service is running on. If interfaces does not match then refer [this](https://github.com/Seagate/cortx-hare/blob/main/README.md#data_iface)
+
+### 2. Test is stuck at starting motr process for first phase
+If the test is stuck at "Starting Motr (phase1, mkfs)... " like shown below, then there is possbility that 
+you have pulled latest code for cortx-motr/dtm0-main branch and rebuilt it, but not rebuilt the hare code again.
```

Comment:
s/cortx-motr/dtm0-main branch/cortx-motr

You don't have to mention the specific branch "dtm0-main", because it is not temporarily on this branch. It will be merged into main later.

Reply:
Ok. got it and agreed.